### PR TITLE
mv assets/ to custom/

### PR DESCRIPTION
**Pull request by samsammurphy**
The assets/ folder contains images and fonts that forked code will need to update (e.g. logo.svg).

Moving it into custom/ folder makes it explicitly that these files are available for custom updates.

The steps I took
1) copy and pasted the assets/ folder into custom/
2) updated src/custom/strings.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# The README cannot be in src/custom but needs to be customizable
+# hence this gitignore (read and update README on github)
+README.md
```
Comment (sebastianovide) [inline .gitignore:3]: the whole purpose of the README is to describe the Geovation Photo project.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,10 @@
-[![Build Status](https://travis-ci.org/Geovation/photos.svg?branch=master)](https://travis-ci.org/Geovation/photos)
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FGeovation%2Fphotos.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FGeovation%2Fphotos?ref=badge_shield)
-[![dependencies Status](https://david-dm.org/geovation/photos/status.svg)](https://david-dm.org/geovation/photos)
-[![Maintainability](https://api.codeclimate.com/v1/badges/f18dd7329321d93cbabb/maintainability)](https://codeclimate.com/github/Geovation/photos/maintainability)
-[![Coverage Status](https://coveralls.io/repos/github/Geovation/photos/badge.svg)](https://coveralls.io/github/Geovation/photos)
-[![npm version](https://badge.fury.io/js/geovation-photos.svg)](https://badge.fury.io/js/geovation-photos)
+# Earthscope photos
```
Comment (sebastianovide) [inline README.md:1]: ?

File: src/custom/strings.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import React from "react";
 
 import CameraRearTwoToneIcon from "@material-ui/icons/CameraRearTwoTone";
-import welcome_logo_svg from "assets/images/logo.svg";
-import welcome_globe_png from "assets/images/globe.png";
-import welcome_chart_png from "assets/images/chart.png";
-import welcome_croud_png from "assets/images/croud.png";
+import welcome_logo_svg from "custom/assets/images/logo.svg";
```
Comment (sebastianovide) [inline src/custom/strings.js:4]: there are two or three places with assets, and when I'ver merged the changes done by Plastic Patrol, they got mixed up in some how. Surerly the cusome images should go in a custom folder, but not sure it is the right one. I need to investigate where the assets should go...

**Comment by samsammurphy** [comment]
Sorry, added more than I wanted to in this PR.

**Pull request closed without merging**